<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OknoLogowania</class>
 <widget class="QWidget" name="OknoLogowania">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>-80</x>
     <y>0</y>
     <width>641</width>
     <height>491</height>
    </rect>
   </property>
   <widget class="QLabel" name="przywitanie">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>201</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 255, 255, 170);
font: 500  &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Witaj,
zaloguj się by rozpocząć</string>
    </property>
   </widget>
   <widget class="QLabel" name="tlo_szare">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>151</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 255);
border-top-left-radius: 55px;
border-bottom-left-radius: 55px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Nazwa_apki">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(255, 255, 255, 200);
font: 500  &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>OfficeHelper</string>
    </property>
   </widget>
   <widget class="QLabel" name="tlo_biale">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>240</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 255);
border-radius: 40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_z_okna">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>130</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(205, 180, 219), stop:1 rgb(255, 255, 255));
color:rgba(0, 0, 0, 240);
padding-bottom:0px;
font: 500  &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Login pracownika</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="haslo">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>220</y>
      <width>181</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 0);
border:none;
border-bottom:2px solid qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(205, 180, 219), stop:1 rgb(255, 255, 255));
color:rgba(0, 0, 0, 240);
padding-bottom:0px;
font: 500  &quot;Futura&quot;;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Hasło pracownika</string>
    </property>
   </widget>
   <widget class="QLabel" name="zaloguj_sie">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>241</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>20</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 500 20pt &quot;Futura&quot;;
color:rgba(0, 0, 0, 200);</string>
    </property>
    <property name="text">
     <string>Zaloguj się</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>310</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(205, 180, 219), stop:1 rgb(189, 224, 254));
	font: 500 12pt &quot;Futura&quot;;
	color:rgba(10, 10, 10, 210);
	border-radius:10px;
}

QPushButton#pushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(215, 190, 229), stop:1 rgb(199, 234, 255));
}

QPushButton#pushButton:pressed{
	padding-left:2px;
	padding-top:2px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(185, 160, 199), stop:1 rgb(169, 204, 225));
}</string>
    </property>
    <property name="text">
     <string>Zaloguj się</string>
    </property>
   </widget>
   <widget class="QLabel" name="build">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>410</y>
      <width>241</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <fontweight>Medium</fontweight>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
color:rgba(0, 0, 0, 50);
font: 500  &quot;Futura&quot;;</string>
    </property>
    <property name="text">
     <string>Build: {wersja}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="zamknij">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#zamknij{
	background-color: rgba(255, 38, 0, 100);
	border-radius:7px;
}

QPushButton#zamknij:hover{
	background-color: rgba(255, 38, 0, 170);
	border-radius:7px;
}


QPushButton#zamknij:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color: rgba(255, 38, 0, 255);
	border-radius:7px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tlo_biale_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255, 255, 255, 70);
border-radius: 47px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="tlo_szare_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0, 0, 0, 100);
border-top-left-radius: 60px;
border-bottom-left-radius: 60px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="blad_logowania">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>250</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 500 8pt &quot;Futura&quot;;
color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>tlo_szare_2</zorder>
   <zorder>tlo_biale_2</zorder>
   <zorder>tlo_szare</zorder>
   <zorder>tlo_biale</zorder>
   <zorder>Nazwa_apki</zorder>
   <zorder>haslo</zorder>
   <zorder>build</zorder>
   <zorder>login_z_okna</zorder>
   <zorder>zaloguj_sie</zorder>
   <zorder>przywitanie</zorder>
   <zorder>pushButton</zorder>
   <zorder>blad_logowania</zorder>
   <zorder>zamknij</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>